frontend/src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { SimpleGrid, Text } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import { useToast, UseToastProps } from '~hooks/useToast'
+import Button from '~components/Button'
+
+import { Toast, ToastProps } from './Toast'
+
+const ToastStateProps: Record<string, ToastProps> = {
+  Warning: {
+    status: 'warning',
+    title: 'Warning',
+    description: 'This is a toast for warning states',
+    isClosable: true,
+    onClose: () => {},
+  },
+  Success: {
+    status: 'success',
+    title: 'Success',
+    description: 'This is a toast for success states',
+    isClosable: true,
+    onClose: () => {},
+  },
+  Error: {
+    status: 'danger',
+    title: 'Error',
+    description: 'This is a toast for error states',
+    isClosable: true,
+    onClose: () => {},
+  },
+}
+export default {
+  title: 'Components/Toast',
+  component: Toast,
+  parameters: { backgrounds: { default: 'light' } },
+  decorators: [
+    // NOTE: The toast component requires this to display properly with theming
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    (Story: Function) => (
```

Comment:
yep will change! i'll change to to `ChakraProvider` because the button requires a theme provider to render the toast correctly but this will remove the need for the comment!